Vindicta.Kunduz/src

```diff
@@ -0,0 +1 @@
+C:/Users/jportilla/OneDrive - ENDAVA/Documents/Arma 3/missions/Vindicta github/Vindicta/src
```

Comment:
Looks strange, we don't need this folder

Reply:
For instance check other folders for maps
https://github.com/Sparker95/Vindicta/tree/development/Vindicta.Tanoa
They don't have src committed.
src folder is actually added to gitignore, I am not sure why git has let you add it.
https://github.com/Sparker95/Vindicta/blob/769c904ad181b5373991a3627159dc9162ea4f6f/.gitignore#L17